Parent_read_points.py

```diff
@@ -0,0 +1,61 @@
+
+def read_file(filename):
+    """
+    Read and return the contents of a file.
+
+    Para:
+        filename (str): The name of the file to be read.
+
+    Returns:
+        str: The contents of the file.
+    """
+    with open(filename, "r") as f:
+        return f.read()
+
+def read_exam_data(exam_names, student_number):
+    """
+    Read exam data from files for a given list of exam names and student number.
+
+    Para:
+        exam_names (list): A list of exam names.
+        student_number (int): 1 for Student1, 2 for Student2.
+
+    Returns:
+        dict: A dictionary where keys are exam names and values are exam data.
+    """
+    exam_data = {}
+    for exam_name in exam_names:
```

Comment:
Nicely structured methods here. But this method looks a lot like it really depends on the data of a student and can't be used anywhere else.
Could it make sense to moved this method somewhere else and restructure it so the dependency to students is more clear?